ref/general/authorization.adoc

```diff
@@ -8,42 +8,79 @@
+To control authorization for front-end resources, such as HTTP calls, you can configure security roles at the servlet level.
+You can specify security roles in the servlet code by configuring the `@ServletSecurity` annotation. Set the `rolesAllowed` attribute within the `@HttpConstraint` or `@HttpMethodConstraint` annotation to the permissible roles, as shown in the following example:
 
-Authorization occurs not only between users and applications, but also between microservices within an application. When one service requests data from another, the requesting service often must be able to demonstrate that it is authorized to access the data. Token-based authorization, link:/docs/ref/general/#sso-config-jwt.html[such as with JSON Web Tokens (JWT)], provides a lightweight way for security controls and security tokens to propagate user identities and roles across different services. Open Liberty supports token-based authorization through the link:/docs/ref/feature/#jwt-1.0.html[MicroProfile JWT feature].
+`@ServletSecurity(value = @HttpConstraint(rolesAllowed = { "user", "admin" }`
 
-== Secure your applications with Open Liberty
+=== Authorizing requests between web services
+
+Authorization occurs not only between users and applications, but also between microservices within an application. When one service requests data from another, the requesting service often must be able to demonstrate that it is authorized to access the data.
+Token-based authorization, link:/docs/ref/general/#sso-config-jwt.html[such as with JSON Web Tokens (JWT)], provides a lightweight way for security controls and security tokens to propagate user identities and roles across different services.
+Open Liberty supports token-based authorization through the link:/docs/ref/feature/#jwt-1.0.html[MicroProfile JWT feature].
+
+=== Configuring authorization with a deployment descriptor file
+
+Although annotations are the most common way to define user roles in microservices, you can also define roles by specifying `<security-constraint>` elements in a `web.xml` deployment descriptor file.
+This configuration might be necessary in situations where you want to keep the security configuration separate from development and apply it at deployment time.
+
+Some applications rely on a combination of declarative annotations and a deployment descriptor file. In these cases, deployment descriptor values supersede annotations.
+Therefore, you can specify security roles in a deployment descriptor file if you want to override security settings or defaults from elsewhere in the application code.
+For an example of `<security-constraint>` configuration in a `web.xml` file, see link:guides/security-intro.html#adding-authentication-and-authorization[the Adding authentication and authorization section of the Securing a web application guide].
+
+== Where to go next
```

Comment:
I think Karen wants us to use "Where to next" based on the feedback she put in https://github.com/OpenLiberty/docs/issues/1304